packages/frontend/src/pages/Template/components/IfThenTemplateStepContent.tsx

```diff
@@ -0,0 +1,87 @@
+import type { IApp } from '@plumber/types'
+
+import { BiQuestionMark } from 'react-icons/bi'
+import { Box, Card, Flex, Icon, Image, Text } from '@chakra-ui/react'
+
+import type { TemplateStep } from '@/graphql/__generated__/graphql'
+
+interface IfThenTemplateStepContentProps {
+  app?: IApp
+  templateSteps: TemplateStep[]
+}
+
+export default function IfThenTemplateStepContent(
+  props: IfThenTemplateStepContentProps,
+) {
+  const { app, templateSteps } = props
+  // sanity check
+  if (!templateSteps || templateSteps.length === 0) {
+    return <></>
+  }
+  const branchNames: string[] = []
+  templateSteps.forEach((templateStep) => {
+    if (templateStep.parameters && templateStep.parameters['branchName']) {
+      branchNames.push(String(templateStep.parameters['branchName']))
+    }
+  })
+  return (
+    <Flex w="100%" flexDir="column">
+      {/* This is for the if then header step */}
+      <Card
+        variant="outline"
+        w="100%"
+        p={4}
+        display="flex"
+        flexDir="row"
+        alignItems="center"
+        columnGap={4}
+        borderTopRadius={8}
+        borderBottomRadius={0}
+      >
+        <Flex
+          borderWidth={1}
+          boxSize={16}
+          borderColor="base.divider.strong"
+          justifyContent="center"
+          alignItems="center"
+          borderRadius="0.25rem"
```

Comment:
prefer not use rem, but use one of chakra's default size 
https://v2.chakra-ui.com/docs/styled-system/theme#border-radius